File: C/lista3 - vetores/exercicio_3.c
#include <conio.h>
#include <stdio.h>

int main() {
	int i, sumGrades = 0;
	float grades[4];
	
	for (i= 0; i< 4; i++) {
		printf("Informe uma nota: ");
		scanf("%f", &grades[i]);
	}
	
	printf("\n");
	
	for (i= 0; i< 4; i++) {
		printf("Nota %d: %.1f \n", (i + 1), grades[i]);
		sumGrades += grades[i];
	}
	
	printf("\n");
	
	printf("Media das notas: %.2f", (float) (sumGrades / 4));
	
	getch();
	return 0;
}

File: Python/lista3 - listas/exercicio_3.py
grades = []

for i in range(4):
    grade = float(input('Informe uma nota: '))
    grades.append(grade)

sumGrades = 0

for i in range(4):
    print(f'Nota {i}: {grades[i]:.2f}')
    sumGrades += grades[i]

print(f'Media das notas: {(sumGrades / 4)}')

